auter.conf

```diff
@@ -19,6 +19,14 @@ PACKAGEMANAGEROPTIONS=""
 # during --prep
 PREDOWNLOADUPDATES="yes"
 
+# Option to only install the downloaded packages. This option is useful if you
```

Comment:
I think we could be more explicit about the use case here?  ie "This option should be used when getting multiple servers to apply the same update list (eg to keep a uat/staging/prod environment in sync), or when you would like to review the updates ahead of applying them.  Not setting this option will result in auter installing the latest available packages at the time apply runs."  Maybe something along those lines?